add-on/chrome/test/mochitest/browser_LoopSidebar.js

```diff
@@ -0,0 +1,20 @@
+/* Any copyright is dedicated to the Public Domain.
+   http://creativecommons.org/publicdomain/zero/1.0/ */
+
+/*
+ * This file contains tests for the LoopSidebar
+ */
+"use strict";
+
+const { LoopSidebar } = Cu.import("chrome://loop/content/modules/LoopSidebar.jsm", {});
+
+add_task(function* test_LoopSidebar_noSidebar() {
+  let sidebar = window.document.getElementById("loop-side-browser");
+  Assert.ok(!sidebar, "there should be no sidebars yet");
+});
+
+add_task(function* test_LoopSidebar_createSidebar() {
+  LoopSidebar.createSidebar(window);
+  let sidebar = window.document.getElementById("loop-side-browser");
+  Assert.ok(sidebar, "there should create a sidebar");
+});
```

Comment:
Once we've loaded the sidebar, we should be able to check the permissions as well, something like:

`Assert.equal(Services.perms.testPermission(uri, "camera"), Services.perms.ALLOW_ACTION, "failure message")`